### ISSUE TITLE
Configuración de Roles del colegio

### ISSUE THREAD

**Issue by facurodriguez** [facurodriguez/MariaAuxiliadora#4]
Se deberá crear los roles de los usuarios que utilizarán el sistema. Estos son:

- **Contador**

- **Docente**

- **Secretario**

La asignación de un rol a un usuario del sistema se realizará en el momento de la creación de usuario. Por el momento se realizará mediante el uso de un `<select></select>`.

La tarea implica además _ocultar_ la existencia del rol Admin existente en el proceso base a la hora de crear un nuevo usuario. @AsmusGerman sugirió que se agregue un atributo booleano a la entidad Role que especifique si el rol es seleccionable o no, lo que me parece una forma correcta y elegante de resolver esto.

**Comment by facurodriguez** [comment]
Hasta el momento la creación de los roles se realiza de forma dinámica cuando la aplicación se ejecuta. Lo que hace es chequear si los roles existen en la base de datos, si no están los crea.

Además, se creó un método que devuelve los roles del colegio filtrando a través del atributo **IsSelectable** de la clase Role.

Cabe mencionar que se creo una migración que debe ser aplicada para poder incorporar esa nueva columna a la tabla AbpRoles.

Si ya se tiene la base de datos creada antes de realizar la migración, una vez que se aplica esa migración se debe modificar de forma manual los valores de la columna IsSelectable de los roles del sistema.

**Comment by AsmusGerman** [comment]
Entonces, se debe realizar un `add-migration` para añadir la nueva columna IsSelectable?

**Comment by facurodriguez** [comment]
No, la migración ya fue creada y subida en uno de los últimos commits. Lo único que se debería ser es un `update-database` o `dotnet ef database update` dependiendo de donde estes trabajando. Siempre teniendo como proyecto predeterminado el de EntityFramework
